src/WordPress/JsonMapper/ObjectWrapper.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace WordPress\JsonMapper;
+
+use BadFunctionCallException;
+use ReflectionClass;
+use UnexpectedValueException;
+use function class_exists;
+use function is_null;
+
+class ObjectWrapper {
```

Comment:
Not a blocker for this PR: Is this class needed? Could we pass objects around instead?

Reply:
It is not useless, that is for sure. I'd happily get rid of it while writing tests to assure no features were lost in the process. This might happen in this PR, with the core tests. Or later, with better test coverage. We'll see.